### DISCUSSION TITLE
ATC Buttons - near TCAS buttons

### DISCUSSION THREAD

**Discussion by GB014934** [flybywiresim/aircraft#4886] [General]
I have the latest development version - updated 09:00 (UK Time) 24/5/2021.

The third button will rotate anti clockwise but I cannot find the click spot where it can be rotated clockwise.
Should it really have the rotate facility as it is a 2 position switch ?  A simple click would do.

This is a very minor inconsistency as I can use the scroll wheel to turn the switch.

Trevor

**Comment by ExampleWasTaken** [comment]
Please attach screenshots of the button your are referring to.
On a side note, if the cursor turns into either an arrow or the `Ø` symbol, you can also scroll the corresponding switch.

**Reply by GB014934** [reply]
Many thanks for your quick reply.

I have included a screen shot of the switch in question.  I can only get the cursor to turn into the circular symbol.
![Screen 20210525](https://user-images.githubusercontent.com/44556994/119469705-8496d980-bd3f-11eb-8992-0e01c8ba21bc.JPG)

**Reply by tracernz** [reply]
You can operate it with your mousewheel (as with the other scroll type knobs). This one should in reality be a two-position switch as you say. Unfortunately not turning all the way on is a model issue, but it should be able to turn all the way off if the xml is correct.

**Comment by averen** [comment]
I believe they are referencing the altitude reporting switch for TCAS. I encountered this issue earlier today as well with the latest development version. It is worth noting that the switch does not move all the way to the ON position either.
![image](https://user-images.githubusercontent.com/31487840/119439577-e58ec500-bcdf-11eb-8f38-59b952c1db07.png)

**Reply by ManyPandas** [reply]
I've been having a similar issue as well. The switch has three positions for some reason, and will not switch over to 'on' even when I try to force it with the mouse.
